policyengine_us/variables/gov/states/la/tax/credits/la_child_tax_credit_refundable.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class la_child_tax_credit_refundable(Variable):
```

Comment:
we need to make a la_child_expense_tax_credit_refundable_eligible file that just compares agi to the 25,000 limit

then la_child_expense_tax_credit_refundable.py which just adds the la_child_expense_tax_credit (just an adds function) - see cdcc PR